pages/account/next-steps.vue

```diff
@@ -69,6 +75,7 @@
     data(){
       return {
         plan: getPlanWithDefault(this.$auth.user.plan_id),
+        seats: this.$route.query.seats || 1
```

Comment:
Once we get the backend set up, we'll be getting this from the user.  Until then, this is a good placeholder.